packages/@aws-cdk/toolkit/lib/actions/deploy/index.ts

```diff
@@ -260,4 +260,12 @@ export interface DeployOptions extends BaseDeployOptions {
    * @deprecated Implement in IoHost instead
    */
   readonly progress?: StackActivityProgress;
+
+  /**
+   * Allows adding CloudWatch log groups to the log monitor via
+   * cloudWatchLogMonitor.setLogGroups();
+   *
+   * @default - not monitoring CloudWatch logs
+   */
+  readonly cloudWatchLogMonitor?: CloudWatchLogEventMonitor;
```

Comment:
Do we really want to make this a public interface? What is the benefit of providing this as a customizable class?
Or would be better off turning this into a boolean option `streamLogs?: boolean`?

Reply:
If we do want this as a public interface, it will have to be an `ICloudWatchLogEventMonitor` etc.